ci/workflows/main-cron.yml

```diff
@@ -211,7 +211,7 @@ steps:
           config: ci/docker-compose.yml
           environment:
             - CODECOV_TOKEN
-    timeout_in_minutes: 22
+    timeout_in_minutes: 25
```

Comment:
Does it mean that UT will spend more time after this PR? Why?

Reply:
Can we revert it after refactor ?